clients/cloud/README.md

```diff
@@ -9,6 +9,8 @@ This directory includes examples of Kafka client applications connecting to [Con
 [![](images/confluent-cli.png)](confluent-cli/)
 [![](images/ruby.png)](ruby/)
 
+[![](images/kotlin.png)](kotlin/)
```

Comment:
I know you like Kotlin but please resize the image -- it's huge ;)

Reply:
Ok